adapter/internal/messaging/notification_listener.go

```diff
@@ -105,6 +105,21 @@ func handleNotification() {
 	logger.LoggerInternalMsg.Infof("handle: deliveries channel closed")
 }
 
+func handleAzureNotification() {
+	for d := range msg.AzureNotificationChannel {
+		logger.LoggerInternalMsg.Infof("[TEST][FEATURE_FLAG_REPLACE_EVENT_HUB] message received for " +
+			"NotificationChannel = " + string(d))
+		var notification msg.EventNotification
+		error := parseNotificationJSONEvent(d, &notification)
+		if error != nil {
+			continue
```

Comment:
Lets handle the errors properly, retrying and etc

Reply:
if there is an error here means there is an issue with the consumed event. So we should just log and continue right ? Retry should happen receiving end before message is given to the channel ?